docs/content/v0.20/assets/README.md

```diff
@@ -0,0 +1,5 @@
+# How to edit these pictures
```

Comment:
Why is there an `assets` subdirectory? If the intent is to group related things together, the name `assets` does not really identify their subject matter.

Reply:
I was undecided if using the name `images` for that folder, but then I thought that could be confusing as well as observers may think of OCI images or something like that. I found other CNCF projects such as Argo CD use the name `assets` (https://github.com/argoproj/argo-cd/tree/master/docs ) - but if that creates confusion we could use the name `diagrams`. Wdyt ?

Reply:
Do we need a subdirectory at all?

Reply:
We can do without it. We'll just have several .md files and .png files under the same directory. I think it's fine.